TAF/config/performance-metrics/configuration.py

```diff
@@ -117,7 +117,7 @@
 
 # Suite: 4_ping_response_time
 # Loop time for sending ping request
-PING_RES_LOOP_TIMES = 100
+PING_RES_LOOP_TIMES = 102
```

Comment:
it would still be 100, but use PING_RES_LOOP_TIMES + 2 in the code
also, the report should still shows 100 times

Reply:
Updated.